sass/civicrm-override/_calender.scss

```diff
@@ -0,0 +1,194 @@
+.ui-datepicker {
+  background: $crm-white;
+  border: 1px solid $datepicker-border-color;
+  border-radius: 0;
+  box-shadow: $box-shadow;
+  min-width: 160px;
+  padding: 10px;
+  width: auto;
+
+  &,
+  * {
+    border-radius: 0;
+    font-family: $font-family-base;
+  }
+
+  * {
+    background: none;
+    box-shadow: none;
+  }
+
+  &,
+  .ui-datepicker-calendar {
+    td[data-handler] a {
+      background: $gray-lighter;
+    }
+
+    td[data-handler] a.ui-state-highlight {
+      border-color: $brand-primary;
+    }
+  }
+}
+
+.ui-datepicker .ui-datepicker-header {
+  border: 0;
+  margin-bottom: 5px;
+
+  .ui-datepicker-next,
+  .ui-datepicker-prev {
+    box-shadow: none;
+    height: 28px;
+    top: 4px;
```

Comment:
I  think `top: xxx` property  is generally used to set top margin for positioned elements other than `position: static`. Are you sure, we need this top property here? As i dont see ` .ui-datepicker-next and .ui-datepicker-prev` don't have position defined.